### ISSUE TITLE
Not Working after update to ubuntu 18.04 - possible fix

### ISSUE THREAD

**Issue by kodi** [MinoMino/minqlx#79]
After I updated my server to ubuntu 18.04, i could no longer run stuff.

So, this is what I did, more or less from my memory, hopefully can help people or be added to the docs.

1) 
ubuntu 18.04 comes with the python 3.6, so we need to add 3.5 as well

```
sudo add-apt-repository ppa:deadsnakes/ppa
sudo apt-get update
sudo apt-get install python3.5 python3.5-dev
```

2) delete old virtual env, and re-create new one, mine virt env folder was named `ENV`
```
rm -rf ENV
python3.5 -m virtualenv ENV
```

3) install required packages

```
source ENV/bin/activate
cd minqlx-plugins/
pip -r requirements.txt
```

It would be great if someone could grab fresh install of 18.04 to verify. I used my memory and bash history, might have missed something

Thanks!

**Comment by em92** [comment]
Hi, kodisha!

Current master version does not support python other than 3.5, but this is already fixed in develop branch. Do this to recompile:
```
git clone https://github.com/MinoMino/minqlx.git
cd minqlx
git checkout develop
make
```

And replace minqlx.so, minqlx.zip with those compiled ones in bin/* directory

**Comment by em92** [comment]
Released version with python 3.6+ support: https://github.com/MinoMino/minqlx/releases/tag/v0.5.2
